Tidied up some irregularities with the documentation

brettansley opened the pull request:
Continuous Deployment and Multiple buildpacks pages were removed
because we have not yet tested them or our testing indicated that there
were some issues with these.

Updated the home page to include a warning against running production apps.

Commit:
Merge branch 'Create_CF_documentation' into move_to_18F_doc_structure

alext commented on CloudFoundry/content/getting-started/concepts.md:
```diff
@@ -75,10 +75,6 @@ cf target -o ORGNAME -s SPACENAME
 All apps need to use a 'buildpack' specific to their language, which sets up dependencies for particular language stacks. There are standard buildpacks for most lanugages, and they will usually be auto-detected by CF. Using the standard buildpacks is strongly encouraged. In the rare case where the buildpack does not get detected correctly, or to use a custom buildpack, it can be specified in the  ...
```
The phrase `Use either the buildpack name` no longer is quite correct now that there's only the one option below.

saliceti committed:
Updated the home page to warn against running prod code

Updated the home page to include a warning that this is not a
production system and that production code and data should not be run
from the platform.

How to test: run hugo in server mode
navigate to localhost/index.html
The welcome page should have text encouraging users not to deploy
production applications.

saliceti committed:
Removed reference to custom buildpack

Removed references to custom buildpacks - we want to encourage everyone
to use platform provided buildpacks.

How to test: run hugo server locally and navigate to localhost/gettingstarted/concepts
Ensure that the documentation doesn’t make reference to custom buildpacks
